geniza/corpus/tests/test_corpus_admin.py

```diff
@@ -370,3 +371,45 @@ def test_collection_display(self, fragment):
         fragment.collection = cul
         frag_admin = FragmentAdmin(model=Fragment, admin_site=admin.site)
         assert frag_admin.collection_display(fragment) == cul
+
+
+class TestHasTranscriptionListFilter:
+    def init_filter(self):
+        # request, params, model, admin_site
+        return HasTranscriptionListFilter(Mock(), {}, Document, DocumentAdmin)
+
+    def test_lookups(self):
+        assert self.init_filter().lookups(Mock(), Mock()) == (
+            ("yes", "Has transcription"),
+            ("no", "No transcription"),
+        )
+
+    @pytest.mark.django_db
+    def test_queryset(self, document, join, typed_texts):
+        filter = self.init_filter()
+
+        # no transcription: all documents should be returned
+        all_docs = Document.objects.all()
+        # no transcription: all documents should be returned
+        with patch.object(filter, "value", return_value="no"):
+            assert filter.queryset(Mock(), all_docs).count() == 2
+        # ahs transcription: no documents should be returned
```

Comment:
_Extremely_ minor, but both of these comments have a typo "ahs transcription".

Reply:
🤦‍♀️ thank you